pages/deployment/linux.mdx

```diff
@@ -0,0 +1,334 @@
+a safe location.
+
+## Backup data
+
+Memgraph uses  `snapshots` and `wal-files`
+folders for [durability](../fundamentals/data-durability), they enable the database and graph to recover in case
+of a crash. The snapshots and wal-files folders are good candidates for backup,
+and they are part of the `/var/lib/memgraph` folder. 
+
+If you want to restore the data, all you need to do is start a new Memgraph
+instance with different data directories by using the `--data-directory` flag
+and pointing it to a backup folder. 
+
+Also, if you are using query modules and custom configuration files, make sure
+to back them up as well. The query modules are stored in the
+`/usr/lib/memgraph`. 
+
+
+### Copy the data persistence folder 
+
+The easiest and most reliable way to backup the graph data is to copy the whole
+`/var/lib/memgraph` folder.  
+
+First, run the following query in Memgraph to lock the data directory in order
+to avoid changes happening during the backup process
+
+```cypher
+LOCK DATA DIRECTORY;
+```
+
+After that, copy the data persistency folder onto your local file system:
+
+```bash
+cp -rp /var/lib/memgraph /path/to/my/local-folder
+```
+
+Notice that the `-p` flag is used to preserve the file ownership and
+permissions.
+
+In the end, unlock the data directly by running the following query in Memgraph:
+
+```cypher
+UNLOCK DATA DIRECTORY;
+```
+
+### Dump database
+
+Another approach is to dump the database into a file. This is useful if you want
+to move the data between different Memgraph versions that might have
+incompatible data formats are used for snapshots or wal files. 
+
+To dump the database, you can use the `mgconsole` CLI tool. Here is an example
+of how to dump the database into a file:
+
+```bash
+echo "DUMP DATABASE;" | mgconsole --output-format=cypherl > data.cypherl
+```
+
+Keep in mind that the `cypherl` format is used to dump the database. Whole graph
+data is stored in the `data.cypherl` file, this means that the import process
+can take a while since the entire graph data is stored in a single file. 
+
+You should prefer copying the data persistency folder when possible over dumping
+the database since it is significantly faster. 
+
+## Restore data
+
+Depending on the backup method you use, you can restore data in a different way.
+If you have copied the data persistency folder, you can simply copy the folder
+back to the custom location and start the Memgraph instance with the
+`--data-directory` flag pointing to the restored folder. 
+
+The backup data folder should be owned by the Memgraph user, so make sure the
+folder is accessible for read and write operations by the Memgraph user. 
+
+Putting the backup folder back to `/var/lib/` path is recommended since you will
+avoid issues with ownership. 
+
+If you have dumped the database, you can import the data back into Memgraph by
+using the `mgconsole` CLI tool or the client library.
```

Comment:
Maybe link this https://github.com/memgraph/mgconsole?tab=readme-ov-file#export--import-into-memgraph?